booking-app/components/src/server/admin.ts

```diff
@@ -255,15 +266,18 @@ export const serverApproveEvent = async (id: string) => {
   serverSendConfirmationEmail(
     id,
     BookingStatusLabel.APPROVED,
-    `This is a confirmation email.`
+    `This is a confirmation email.`,
+    guestEmail
   );
 
   // for Samantha
   serverSendBookingDetailEmail(
     id,
     getApprovalCcEmail(process.env.NEXT_PUBLIC_BRANCH_NAME),
     `This is a confirmation email.`,
-    BookingStatusLabel.APPROVED
+    BookingStatusLabel.APPROVED,
+    undefined,
```

Comment:
Instead of using `undefined` for optional parameters, it might be clearer to group them into a single options object. It helps readability and makes it easier to manage future changes or additions.

ex:
```
// current
serverSendBookingDetailEmail(id, email, headerMessage, status, undefined, guestEmail);

// suggested
serverSendBookingDetailEmail({
  calendarEventId: id,
  email,
  headerMessage,
  status,
  replyTo: guestEmail,
});
```

Reply:
Sure! I'll refactor this in this PR. I guess I just want to keep the changes as small as possible at first 🤣